dev/hal/inc/Tft.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file Tft.h
+ * @author noura36
+ * @brief
+ * @version 0.1
+ * @date 2024-08-20
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+#ifndef DEV_HAL_INC_TFT_H_
+#define DEV_HAL_INC_TFT_H_
+using namespace stm32::dev::mcal::pin; 
+using namespace stm32::dev::mcal::gpio;
+using namespace stm32::dev::mcal::spi;
+namespace stm32 {
+namespace dev {
+namespace hal {
+namespace tft {
+enum TftCommand {
+    kSLPOUT   = 0x11,     //  Sleep out & booster on
+    kCASET    = 0x2A,     //  Column address set
+    kRASET    = 0x2B,     //  Row address set
+    kRAWMR    = 0x2C,     //  Memory write
+    kCOLMOD   = 0x3A,     //  Interface pixel format
+    kCOLMOD_P = 0x05,     //  RGB 5-6-5
```

Comment:
The color modes like `RGB565` or `RGB444` are not a commads they are a parameter to `kCOLMOD` comand so make them in a separate enum